scripts/biomedical/NCBI_ClinVar/scripts/format_clinvar_and_position.py

```diff
@@ -0,0 +1,1209 @@
+    w.write('chrom', f'chr{line[0]}')
+    w.write('position', line[1])
+    w.write('referenceAllele', line[3])
+    w.write('alternativeAllele', line[4])
+    w.write('hg19GenomicPosition', line[1])
+
+    writeEntry2('clinVarAlleleID', dict_info, 'ALLELEID', w, True)
+
+    writeClinicalDiseases(dict_info, 'CLNDISDB', w)
+    writeClinicalDiseases(dict_info, 'CLNDISDBINCL', w)
+    writeClinicalDiseases(dict_info, 'ONCDISDB', w)
+    writeClinicalDiseases(dict_info, 'SCIDISDB', w)
+
+    write_disease_name(dict_info, 'CLNDN', w)
+    write_disease_name(dict_info, 'CLNDNINCL', w)
+    write_disease_name(dict_info, 'ONCDN', w)
+    write_disease_name(dict_info, 'SCIDN', w)
+    writeEntry2('hgvsNomenclature', dict_info, 'CLNHGVS', w, True)
+
+    write_review_status(dict_info, 'CLNREVSTAT', 'clinVarGermlineReviewStatus',
+                        w)
+    writeClinSig(dict_info, 'CLNSIG', w)
+    writeClinSig(dict_info, 'CLNSIGINCL', w)
+
+    writeEntry('geneticVariantClass', convertToClassEnum(dict_info, 'CLNVC'), w,
+               False)
+    write_variant_database_ids(dict_info, 'CLNVI', w)
+    write_sequence_ontology('sequenceOntologyID', dict_info, 'CLNVCSO', w)
+    writeGeneInfo(dict_info, 'GENEINFO', w, ASSEMBLY_HG38)
+    writeSeqOntologyFunctionalType(dict_info, 'MC', w)
+
+    writeOrigin(dict_info, 'ORIGIN', w)
+
+    writeEntry2('dbVarID', dict_info, 'DBVARID', w, True)
+
+    write_clin_sig_category(dict_info, 'ONC',
+                            'clinicalSignificanceCategory_onc', w)
+
+    write_review_status(dict_info, 'ONCREVSTAT',
+                        'clinVarOncogenicityReviewStatus', w)
+
+    write_clin_sig_category(dict_info, 'SCI',
+                            'clinicalSignificanceCategory_sic', w)
+
+    write_review_status(dict_info, 'SCIREVSTAT', 'clinVarSomaticReviewStatus',
+                        w)
+    #print("line", '\n', line, '\n', "CURRENT_ROW", CURRENT_ROW)
+
+    return w, dict_info, dcid, rsID, CURRENT_ROW
+
+
+def writeNode_POS(line, count, ASSEMBLY_HG19, start_time):
+    w = WriteToCsv()
+    parse_clinvar_pos_row(line, w, None)
+    # print("POSline", line, '\n', CURRENT_ROW)
+    #w.write('\n')
+    w.write_to_file(['clinvar_pos_only'])
+    w.close()
+    count += 1
+    if count % 100000 == 0:  # start writing to new output file
+        time_running = time.time() - start_time
+        logging.info(f"{count} lines processed: {time_running:.2f} second")
+    return (count)
+
+
+def parse_clinvar_pos_row(line, w, curr_row=None):
+    global CURRENT_ROW
+    if curr_row:
+        CURRENT_ROW = copy.deepcopy(curr_row)
+
+    l = line[7].split(';')
+    dict_info = {}
+    dcid = 'bio/VariationID_' + line[2]
+    for item in l:
+        entry = item.split('=')
+        dict_info[entry[0]] = entry[1]
+    if 'RS' in dict_info:
+        dcid = 'bio/rs' + dict_info['RS']
+        rsID = dict_info['RS']
+        w.write('dcid', f"dcid:{dcid}")
```

Comment:
do not include dcid prefix for any dcids! We should initialize all nodes in the tmcf. Any nodes referenced should be initialized in the tMCF and then have the tMCF reference that node for that property within the tMCF. When we are initializing the dcid, we do not include the prefix for the dcid. Please fix this throughout the code. If the tMCfs are not set up this way to reference all nodes referenced in this import then update the tMCFs too.